.changeset/slick-trains-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+fix(sync): add missing live content collections check for generating content collections types
```

Comment:
Can you reword the changeset to make it more user-friendly? The changeset is meant for our end-users, not us developers. So you need to reword it in a way that users understand what it's been fixed.

Reply:
Thanks for the suggestion!